docs/en/toolkit/learning-resources/tutorial.md

```diff
@@ -826,8 +844,10 @@ For more information on reviewing media in Shotgun, see the [official documentat
 
 Congratulations, you're done! Hopefully this tutorial has given you a starting point for building your own custom pipeline using the Shotgun integrations. You should have an understanding of how to extend the default integrations to meet the specific needs of your studio.
 
-If there are features or workflows that you feel are outside of the default integrations, then you can always write your own apps. [Here is an excellent document](https://support.shotgunsoftware.com/entries/95440137) to help you get started writing your first app.
+To learn more about developing your own apps, check out the [App Development Guide](https://support.shotgunsoftware.com/hc/en-us/articles/219033158-Developing-Apps).
 
-As always, if you have additional questions about this tutorial or about Shotgun or the Toolkit platform in general, feel free to [submit a ticket](https://support.shotgunsoftware.com/hc/en-us/requests/new).
+Ask questions and learn how other studios are using Toolkit over at the [shotgun-dev Google Group](https://groups.google.com/a/shotgunsoftware.com/forum/#!forum/shotgun-dev).
 
+If there are features or workflows that you feel are outside of the default integrations, then you can always write your own apps. [Here is an excellent document](https://support.shotgunsoftware.com/entries/95440137) to help you get started writing your first app.
 
```

Comment:
I think this is reduntant with the first bullet point, no? Points to the same doc (despite the URLs being different).

Reply:
yep, removed.